native/src/components/MalteHelpFormOffer.tsx

```diff
@@ -113,58 +114,60 @@ const MalteHelpFormOffer = ({
   })
 
   return (
-    <Container>
-      <InformationRow>
-        <Icon Icon={SupportIcon} />
-        <InformationText>{t('supportNote')}</InformationText>
-      </InformationRow>
-      <InformationRow>
-        <Icon Icon={SecurityIcon} />
-        <InformationText>{t('securityNote')}</InformationText>
-      </InformationRow>
-
-      <FormInput name='name' title={t('name')} control={control} rules={{ required: true }} />
-      <FormInput name='roomNumber' title={t('roomNumber')} control={control} showOptional />
-
-      <View>
-        <InputTitle>{t('howToBeContacted')}</InputTitle>
-        <FormRadioButtons
-          name='contactChannel'
+    <KeyboardAwareScrollView extraHeight={200}>
```

Comment:
```suggestion
    <KeyboardAwareScrollView extraScrollHeight={dimensions.headerHeight + useSafeAreaInsets().top}>
```
Please avoid using magic numbers. To calculate the proper scroll height we should use the header height and the height of the safe area inset (notches etc) of the particular device.
Tested on iPhone15 and iPhone SE3